lib/open_api_spex/cast/utils.ex

```diff
@@ -47,4 +46,11 @@ defmodule OpenApiSpex.Cast.Utils do
   end
 
   def check_required_fields(_ctx, _acc), do: :ok
+
+  defp get_property_schema(ctx, property_name) do
+    case ctx.schema.properties[property_name] do
+      schema = %_{} -> OpenApiSpex.resolve_schema(schema, ctx.schemas)
+      _ -> nil
```

Comment:
When does execution reach this line? When there's a value in the `required` list that doesn't correspond to a key in the `properties` map?